addons/PerformanceMonitor/Core/fn_serverInit.sqf

```diff
@@ -19,8 +19,8 @@
 	PERF_INFO_RECEIVE,
 	{
 		_this select 0 params["_sender", "_fps"];
-		
-		"logger" callExtension format ["%1;%2;", _sender, _fps];
+		private _separator = toString [9]; // tabulator
+		"logger" callExtension format ["%1%2%3", _sender, _separator, _fps];
```

Comment:
What's the rationale/advantage of using Tabs here?

Reply:
> What's the rationale/advantage of using Tabs here?

To get it parsable when playernames contains spaces or semicolons, etc.

Reply:
Yikes! Players include semicolons in their names? xD
But okay then that makes sense :+1: